### PR TITLE
Metric Monitoring: Set Prometheus Version Tag

### LINKED ISSUE #383: Pull too many prom/prometheus images
I'm following `develop` branch now. When I start with the quick-start guide, Clipper pulled too many prom/prometheus images like below,
```
$ docker images
REPOSITORY                    TAG                 IMAGE ID            CREATED             SIZE
prom/prometheus               master              8c9f2c21a127        24 hours ago        112MB
clipper/management_frontend   develop             afebff8f8e71        5 days ago          1.71GB
clipper/query_frontend        develop             a8fe57bb1ae5        5 days ago          1.71GB
redis                         alpine              cb1ec54b370d        5 days ago          27.1MB
prom/prometheus               latest              c8ecf7c719c1        11 days ago         112MB
prom/prometheus               v2.1.0              c8ecf7c719c1        11 days ago         112MB
clipper/frontend-exporter     develop             ca68bf086fe6        12 days ago         701MB
prom/prometheus               v2.0.0              67141fa03496        2 months ago        80.2MB
prom/prometheus               v2.0.0-rc.3         4ec131e5d3fa        2 months ago        80.2MB
prom/prometheus               v1.8.2              b5ecb4cdba16        2 months ago        75.4MB
prom/prometheus               v2.0.0-rc.2         5b7b908224b6        3 months ago        80.1MB
prom/prometheus               v1.8.1              c4768ad32fa2        3 months ago        75.4MB
prom/prometheus               v2.0.0-rc.1         74e07b6d864b        3 months ago        80.1MB
prom/prometheus               v1.8.0              03f045a7680f        3 months ago        75.4MB
prom/prometheus               v2.0.0-rc.0         f1cc14972265        3 months ago        79.9MB
clipper/management_frontend   0.2.0               6b8f015eeee7        3 months ago        1.73GB
clipper/query_frontend        0.2.0               7ce07e68fe00        3 months ago        1.73GB
prom/prometheus               v1.7.2              4ede871a94fd        4 months ago        74.6MB
prom/prometheus               v2.0.0-beta.5       8408d52e2ebd        4 months ago        81.2MB
prom/prometheus               v2.0.0-beta.4       b95dc65b9535        4 months ago        81.5MB
prom/prometheus               v2.0.0-beta.3       bb3ac1cdb4b7        4 months ago        81.5MB
prom/prometheus               v2.0.0-beta.2       fa66f531fd09        5 months ago        81.3MB
prom/prometheus               v2.0.0-beta.1       cdade62bdb17        5 months ago        81.3MB
prom/prometheus               v2.0.0-beta.0       1281ababcc1d        6 months ago        80.9MB
prom/prometheus               v2.0.0-alpha.3      420515b01603        7 months ago        75.9MB
prom/prometheus               v1.7.1              4da113bb6ae3        7 months ago        74.5MB
prom/prometheus               v1.7.0              bb25f19f9142        7 months ago        74.5MB
prom/prometheus               v2.0.0-alpha.2      c75bda08f935        8 months ago        72.6MB
prom/prometheus               v1.6.3              b0195cb1a666        8 months ago        73.6MB
prom/prometheus               v2.0.0-alpha.1      b7a19a48e93e        8 months ago        72.6MB
prom/prometheus               v1.6.2              f42e80cfa4f9        8 months ago        73.6MB
prom/prometheus               v1.5.3              7a066756536d        8 months ago        73.3MB
prom/prometheus               v1.6.1              449fb4e4145e        9 months ago        75.3MB
prom/prometheus               v1.6.0              c351d5f42485        9 months ago        75.3MB
prom/prometheus               v2.0.0-alpha.0      f48a9cc3247d        9 months ago        73.9MB
prom/prometheus               v1.5.2              30ee439fbdcf        11 months ago       79.6MB
prom/prometheus               v1.5.1              b0c9d3304305        11 months ago       79.6MB
prom/prometheus               v1.5.0              b028e7b4404b        12 months ago       79.6MB
prom/prometheus               v1.4.1              bdeacb538ef9        14 months ago       79.2MB
prom/prometheus               v1.4.0              0b284a53c234        14 months ago       79.2MB
prom/prometheus               v1.3.1              91d1fc39c411        15 months ago       75.3MB
prom/prometheus               v1.2.3              d7eb04ee02d2        15 months ago       39.6MB
prom/prometheus               v1.3.0              598b21494810        15 months ago       75MB
prom/prometheus               v1.2.2              24121f403aba        15 months ago       39.3MB
prom/prometheus               v1.3.0-beta.0       f1c5257dfd19        15 months ago       75MB
prom/prometheus               v1.2.1              18eb3e291484        15 months ago       39.2MB
prom/prometheus               v1.2.0              cd435e2eff75        16 months ago       48MB
prom/prometheus               v1.1.3              324cf4f3e246        16 months ago       45.5MB
prom/prometheus               v1.1.2              695eb0489cd5        16 months ago       45.5MB
prom/prometheus               v1.1.1              56475ab7cdc3        17 months ago       45.5MB
prom/prometheus               v1.1.0              cd790ef081d7        17 months ago       45.5MB
prom/prometheus               v1.0.2              fda0e1a33eee        17 months ago       43.2MB
prom/prometheus               v1.0.1              62b473b89d8d        18 months ago       43.2MB
prom/prometheus               v1.0.0              e180d8c8c83c        18 months ago       43.2MB
prom/prometheus               v1.0.0-rc.0         08ce694d8fe3        18 months ago       43.2MB
prom/prometheus               0.20.0              93b75d10220c        19 months ago       43.2MB
prom/prometheus               0.19.3              b677f07d716b        19 months ago       43.2MB
prom/prometheus               0.19.2              fbc0b6ce5d5c        20 months ago       43.2MB
prom/prometheus               0.19.1              1bcda31a143f        20 months ago       43.2MB
prom/prometheus               0.19.0              0d919667cfdc        20 months ago       43.2MB
prom/prometheus               0.18.0              9c797c0e36dd        21 months ago       58.5MB
prom/prometheus               0.18.0rc1           cdad9758f5b9        22 months ago       58.5MB
prom/prometheus               0.17.0              4537d00a5507        23 months ago       58.1MB
prom/prometheus               0.17.0rc2           c3f6e83da208        24 months ago       58.1MB
prom/prometheus               0.17.0rc1           760ff6a65384        2 years ago         58MB
prom/prometheus               0.16.2              67c72ca3d08f        2 years ago         59.8MB
prom/prometheus               0.16                65bcef6543b4        2 years ago         59.8MB
prom/prometheus               0.15                05794e5d9f99        2 years ago         48.9MB
prom/prometheus               0.15.1              4e02c813ef4f        2 years ago         48.9MB
prom/prometheus               0.15.0              74b820369d53        2 years ago         48.6MB
prom/prometheus               0.16.0rc1           cd7c124df82d        2 years ago         50.9MB
prom/prometheus               0.16.0rc2           330281372033        2 years ago         51MB
prom/prometheus               0.16.0              9989c05b3f9e        2 years ago         58MB
prom/prometheus               0.16.1              8ddd10fb6089        2 years ago         58MB
```

### PR DESCRIPTION
**Pull request by simon-mo**
Fixes #383

Also sneak in the new example for monitoring.

### CODE REVIEW

File: clipper_admin/clipper_admin/docker/docker_metric_utils.py [hunk 1]
```diff
@@ -4,6 +4,8 @@
 import os
 from ..version import __version__
 
+PROM_VERSION = "master"
```
Comment (dcrankshaw) [inline clipper_admin/clipper_admin/docker/docker_metric_utils.py]: Let's use the latest release (`v2.1.0`) instead of master

**Review by dcrankshaw — CHANGES_REQUESTED** [review]
Just change the tag to the latest released version and this is good to go

File: clipper_admin/clipper_admin/docker/docker_metric_utils.py (hunk 1 above)
Comment (simon-mo) [reply clipper_admin/clipper_admin/docker/docker_metric_utils.py]: Haha that was my in first commit but I changed it, I'll update it

**Review by dcrankshaw — APPROVED** [review]
lgtm

**Pull request merged**